### PR TITLE
Fin

### PR DESCRIPTION
**Pull request by Outta-my-way-burrocrat**

### CODE REVIEW

**Comment by Outta-my-way-burrocrat** [comment]
The rebel interface bug is caused by `$CULTURE$` or `$CULTURE_ADJ$` code in _title localizations

Example

https://github.com/settintotrieste/Victoria-2-Cold-War-Enhancement-Mod-CWE/blob/cf4f32203dc2d77917e77a39af5666f651df8c32/CWE/localisation/cwe_politics.csv#L171

**Comment by Outta-my-way-burrocrat** [comment]
I got a mystery crash at one point a month ago but I can't replicate it now; could some folks from Discord test the mod after I'm gone and make sure everything is ok? I'll test until tomorrow or Sept. 29th and send a fix if I can.

File: CWE/history/units/WSA_oob.txt
[the remark this answers is not in the document]
Comment (Outta-my-way-burrocrat) [reply CWE/history/units/WSA_oob.txt:1]: Gonna test this with vanilla, most of their non-existing countries have oob files.

**Comment by settintotrieste** [comment]
> The rebel interface bug is caused by `$CULTURE$` or `$CULTURE_ADJ$` code in _title localizations
> 
> Example
> 
> https://github.com/settintotrieste/Victoria-2-Cold-War-Enhancement-Mod-CWE/blob/cf4f32203dc2d77917e77a39af5666f651df8c32/CWE/localisation/cwe_politics.csv#L171

Alright.

**Comment by settintotrieste** [comment]
> I got a mystery crash at one point a month ago but I can't replicate it now; could some folks from Discord test the mod after I'm gone and make sure everything is ok? I'll test until tomorrow or Sept. 29th and send a fix if I can.

Sure. I’ll ask the discord people to be on the look out for any crashes and issues and I’ll try to fix them if it’s simple.

**Comment by Outta-my-way-burrocrat** [comment]
>Alright. Perhaps it can be renamed to Minority Rights Militants instead.
Most of the rebel _titles have duplicate locs and similar issues.

**Comment by Outta-my-way-burrocrat** [comment]
Don't think I can add the expanded Afghan civil war mechanic in time but I'll try.

**Comment by settintotrieste** [comment]
> > Alright. Perhaps it can be renamed to Minority Rights Militants instead.
> 
> Most of the rebel _titles have duplicate locs and similar issues.

I see. Changing it would be difficult so if it’s not game breaking then it can be left as it.

**Comment by Outta-my-way-burrocrat** [comment]
>I see. Changing it would be difficult so if it’s not game breaking then it can be left as it.

I've already done it but I fear it may have caused the mystery crash so it needs to be tested more.

**Comment by settintotrieste** [comment]
> > I see. Changing it would be difficult so if it’s not game breaking then it can be left as it.
> 
> I've already done it but I fear it may have caused the mystery crash so it needs to be tested more.

Ah okay

**Pull request merged**